src/lib/CollisionPrevention/collisionprevention_params.c

```diff
@@ -73,4 +73,14 @@ PARAM_DEFINE_FLOAT(MPC_COL_PREV_DLY, 0.4f);
  * @unit [deg]
  * @group Multicopter Position Control
  */
-PARAM_DEFINE_FLOAT(MPC_COL_PREV_CNG, 30.f);
+PARAM_DEFINE_FLOAT(MPC_CP_GUIDE_ANG, 30.f);
```

Comment:
Guide doesn't feel quite right. I guess you're thinking of it like when you go 10 pin bowling and put up the  *guide rails* to prevent the ball going into the gutter? I can't think of a better word though.

Reply:
Yes, I was struggling a bit with that one too. I was thinking of how to explain best what influence the parameter has. And what that feature does, is help you navigate tight spaces or guide/ adjust your input slightly to fit the obstacle map. But could not come up with anything better either...

Reply:
Yep. Perhaps "adjust" is better - so MPC_CP_ADJUST_ANG. Not sure enough to say we should change it.